addons/html_editor/static/src/components/history_dialog/history_dialog.js

```diff
@@ -45,15 +45,19 @@ export class HistoryDialog extends Component {
         revisionContent: null,
         revisionComparison: null,
         revisionId: null,
-        revisionLoading: true,
-        cssMaxHeight: 400,
+        isLoading: false,
```

Comment:
This should be true by default to allow  ` loadBundle("html_editor.assets_history_diff");` during init if necessary.

see #233584